### ISSUE TITLE
Error editing issue with existing fix version

### ISSUE THREAD

**Issue by nrayapati** [jenkinsci/jira-steps-plugin#3]
There are few issues with `jiraEditIssue` step in the below example: 

```groovy
node {
  stage('Edit Issues') {
    def searchResults = jiraJqlSearch jql: "project = TEST AND issuekey = 'TEST-1'"
    def issues = searchResults.data.issues
    for (i = 0; i <issues.size(); i++) {
      def result = jiraGetIssue idOrKey: issues[i].key
      def newVersion = jiraNewVersion version: [name: "new-fix-version-1.0",
                                                project: "TEST"]
      def fixVersions = result.data.fields.fixVersions << newVersion.data
      def testIssue = [fields: [summary: "Test Issue summary",
                                description: "Test description",
                                project: [id : 10000],
                                issuetype: [id : 10000],
                                fixVersions: fixVersions]]
      response = jiraEditIssue idOrKey: issues[i].key, issue: testIssue
    }
  }
}
```

Error while executing above:
```groovy
Error Message: {"errorMessages":[],"errors":{"fixVersions":"expected 'id' property to be a string"}}
```

1. Need to fix above error.
2. `testIssue` variable above really doesn't require other fields than just fixversions.
3. meta is being appended every time we update the same issue. 
4. Also documentation needs an update.

**Comment by nrayapati** [comment]
Issued a pull request for this issue https://github.com/jenkinsci/jira-steps-plugin/pull/4

**Comment by nrayapati** [comment]
Merged #4 and Closing this issue.
